app/_data/docs_nav_gateway_3.8.x.yml

```
@@ -737,6 +737,8 @@ items:
         src: /gateway/reference/configuration/configuration-3.7.x
       - text: Injecting Nginx Directives
         url: /reference/nginx-directives
+      - text: Brotli Compression
```

```suggestion
      - text: Improve performance with Brotli compression
```
Just to give users an indication of what Brotli compression can do for them here

---

For the location in the nav, maybe this could go in Kong in Production? Maybe we could rename "Performance testing" to "Performance" and add it there?